src/main/kotlin/exercise2/task1/FindPairOfHighestSum.kt

```diff
@@ -18,7 +18,10 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPair(): Pair<Int, Int> {
-    TODO("Implement me!!")
+    val prvi = this.sorted().get(this.lastIndex)
+    val drugi = this.sorted().get(this.lastIndex-1)
```

Comment:
There is no need to `sort` the array twice. Please name the variables in English, as that is a widely accepted convention.